bootstrap-salt.sh

```diff
@@ -3733,29 +3733,31 @@ install_amazon_linux_ami_2010_git_deps() {
 }
 
 install_amazon_linux_ami_deps() {
-    # According to http://aws.amazon.com/amazon-linux-ami/faqs/#epel we should
+    # enable the EPEL repo
+    /usr/bin/yum-config-manager --enable epel || return 1
 
-    # enable the EPEL 6 repo
-    if [ "$CPU_ARCH_L" = "i686" ]; then
-        EPEL_ARCH="i386"
-    else
-        EPEL_ARCH=$CPU_ARCH_L
-    fi
-    rpm -Uvh --force "http://mirrors.kernel.org/fedora-epel/6/${EPEL_ARCH}/epel-release-6-8.noarch.rpm" || return 1
+    # exclude Salt and ZeroMQ packages from EPEL
+    /usr/bin/yum-config-manager epel --setopt "epel.exclude=zeromq* salt* python-zmq*" --save || return 1
 
-    __REPO_FILENAME="saltstack-salt-epel-6.repo"
+    __REPO_FILENAME="saltstack-repo.repo"
 
     if [ ! -s "/etc/yum.repos.d/${__REPO_FILENAME}" ]; then
-        echoinfo "Adding SaltStack's COPR repository"
-        __fetch_url /etc/yum.repos.d/${__REPO_FILENAME} \
-            "http://copr.fedoraproject.org/coprs/saltstack/salt/repo/epel-6/${__REPO_FILENAME}" || return 1
+      cat <<_eof > "/etc/yum.repos.d/${__REPO_FILENAME}"
+[saltstack-repo]
+disabled=False
+name=SaltStack repo for RHEL/CentOS 6
+gpgcheck=1
+gpgkey=https://repo.saltstack.com/yum/rhel6/SALTSTACK-GPG-KEY.pub
```

Comment:
@shawnbutts, these URLs should be `https://repo.saltstack.com/yum/redhat/6/\$basearch/$STABLE_REV/`.